gen/elerning_mu_ff

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use File::Basename;
+
+our $SERVICE_NAME = basename($0);
+our $PROTOCOL_VERSION = "3.0.1";
+my $SCRIPT_VERSION = "3.0.2";
+
+perunServicesInit::init;
+my $DIRECTORY = perunServicesInit::getDirectory;
+my $data = perunServicesInit::getHashedHierarchicalData;
+
+#Constants
+our $A_USER_FACILITY_LOGIN;          *A_USER_FACILITY_LOGIN =            \'urn:perun:user:attribute-def:def:login-namespace:mu';
+our $A_USER_FIRST_NAME;              *A_USER_FIRST_NAME =                \'urn:perun:user:attribute-def:core:firstName';
+our $A_USER_LAST_NAME;               *A_USER_LAST_NAME =                 \'urn:perun:user:attribute-def:core:lastName';
+our $A_USER_STATUS;                  *A_USER_STATUS =                    \'urn:perun:member:attribute-def:core:status';
+
+our $STATUS_VALID;                   *STATUS_VALID =                     \'VALID';
+
+
+my $fileName = "$DIRECTORY/$::SERVICE_NAME";
+
+my $users;
+#####################################
+
+foreach my $resourceId ($data->getResourceIds()) {
+	foreach my $memberId ($data->getMemberIdsForResource(resource => $resourceId)) {
+		my $memberStatus = $data->getMemberAttributeValue(member=> $memberId, attrName=>$A_USER_STATUS);
+		my $memberLogin = $data->getMemberAttributeValue(member=> $memberId, attrName=>$A_USER_FACILITY_LOGIN);
```

Comment:
This should be also user attribute, so use `getUserAttributeValue()`. And probably change variable `$A_USER_FACILITY_LOGIN` to `$A_USER_LOGIN`, since its not bound to virtual user-facility attribute, but defined user attribute from specific namespace (including its definition above).